Git process.md

```diff
@@ -29,3 +29,69 @@ While the application is still not deployed on a production server, you can omit
 **Make small commits, following the Single Responsibility Principle in Git.** This makes commits easier to review when making pull requests, and it's easier to notice what's going on when something goes wrong.
 
 **Don't use `git add .`** Review what you're adding to your repo—this is the #1 cause of making unwanted changes.
+
+### Github Pull Request Descriptions
+
+It's essential to write a good Pull Request description. Reviewers will usually read it before
+looking at the diff, so make sure it gives them enough context, so they know what they are looking at.
+
+Another purpose of the good description is to provide the documentation for a future reference.
```

Comment:
I find this a little bit problematic. It's definitely good to write descriptions to inform reviewers, help someone dealing with a specific issue in the future etc., but it is highly unlikely that months later someone will go over old pull requests to read the descriptions, nor is it realistic to expect for example a new developer on the project to get familiar with the project in this way.
Basically, I think that any important content in descriptions should be moved to some centralized documentation (like readme) once the PR is merged (immediately so it's not forgotten).

Reply:
I've been part of a team where we were all writing PR descriptions in this way. When I joined the team, I was "git blaming" lines of code all the time, which lead me to a PR where the line was introduced. Understanding the context in which the line was written helped me understand the line better. This way, I was able to ask other team members more informed questions.

Pull Requests descriptions are a perfect way to investigate how the codebase developed and why it developed the way it did. They are a documentation of not only the current codebase but the evolution behind it.

Centralized documentation is focused on the current state of the app. PR descriptions are documentation on how it evolved to the current state.

They are not a replacement for centralized documentation, but there is an overlap between questions they both answer.

Reply:
Yes, I agree descriptions can be helpful in the scenario you mention, but I meant that if the descriptions contained something important enough that it should be a requirement to add this to the centralized documentation instead of treating it as unnecessary because "it has already been written somewhere".

Reply:
I see, I'm not saying centralized documentation is unnecessary.

Reply:
I know, I thought it would be useful to expand this part to include this extra requirement of maintaining centralized documentation as a reminder.

Reply:
I agree with @cukoreva, it doesn't hurt to mention the requirement of maintaining centralized documentation here

Reply:
@cukoreva, please provide a suggestion on how to phrase this

Reply:
@melcha I see Eva is on vacation this week. Could you please provide a suggestion on how to phrase the requirement of maintaining centralized documentation?

Reply:
@tomazzlender just add a sentence as a reminder that any important information from the description should also be added to the "main" documentation in the project as a central reference point, I wouldn't make this too elaborate as this is not the main topic of your text.

Reply:
@cukoreva 👍 please approve